### DISCUSSION TITLE
Outline client installation error

### DISCUSSION THREAD

**Discussion by borat3** [OutlineFoundation/outline-sdk#173] [Q&A]
Hello friends! I am writing this text through Google Translator, so my words may look crooked. I installed the Outline manager fine, but I have a problem installing the Outline client :(

ERROR MESSAGE: 

---------------------------
Sorry, we could not configure your system to connect to Outline. Please try running the installer again. If you still cannot install Outline, please get in touch with us and let us know that the TAP device failed to install with error code error.
---------------------------

I tried installing different versions, looked at all the solutions on Github and YouTube, but it didn’t help.
My operating system is Windows 11 Home

I tried different options, including with TAP. If you can help, I will be grateful

![photo_2024-01-28_13-46-11](https://github.com/Jigsaw-Code/outline-sdk/assets/111078622/b21ce9ce-5979-490a-9479-c57a12eaf6cd)
![photo_2024-01-28_13-47-38](https://github.com/Jigsaw-Code/outline-sdk/assets/111078622/156548d2-7a7f-4aac-9b64-a94e4def7cab)

**Comment by fortuna** [comment]
Please [file an issue on outline-client](https://github.com/Jigsaw-Code/outline-client/issues/new/choose) instead, or contact https://support.getoutline.org/
